### PR TITLE
Clean tab

### PR DESCRIPTION
**Pull request by sm-ll**
Fixed the tab/overlapping text in ipad view.
Updated to correct copy and flow for CV uploading, separating onboarding and advisor upload.

### CODE REVIEW

File: features/step_definitions/action_plan_steps.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Given(/^I assign a new action plan task to them$/) do
   click_on @client.name
-  click_on "#{@client.first_name}'s Action Plan"
```
Comment (sm-ll) [inline features/step_definitions/action_plan_steps.rb]: Ah yeah, I remember changing this. Nice.

**Comment by sm-ll** [comment]
Happy for it to be merged

**Comment by sm-ll** [comment]
[Fix overlapping bug in mobile view tabs](https://trello.com/c/qyL2q6Jc/29-fix-overlapping-bug-in-mobile-view-tabs)

**Comment by sm-ll** [comment]
[As a client I want to be clear to upload a CV through my onboarding](https://trello.com/c/UuM853BX/50-as-a-client-i-want-to-be-clear-to-upload-a-cv-through-my-onboarding)

**Comment by sm-ll** [comment]
[As an advisor I want to be clear to upload multiple file types, cv, draft, cover letter, note etc](https://trello.com/c/XWqJmGAt/51-as-an-advisor-i-want-to-be-clear-to-upload-multiple-file-types-cv-draft-cover-letter-note-etc)

**Comment by pezholio** [comment]
Hrm. Not sure why the coverage has gone down so much

**Pull request merged**